corehq/apps/translations/management/commands/translate_po_files.py

```
@@ -0,0 +1,672 @@
+        translation_obj_map = {}
+        for index, message_obj in enumerate(translations):
+            translation_obj_map[str(index)] = message_obj
+        return translation_obj_map
+
+    def format_input(self, msg_id_batch):
+        batch_dict = {}
+        for index, message_obj in msg_id_batch.items():
+            batch_dict[index] = message_obj.msgid
+        return json.dumps(batch_dict)
+
+    def create_batches(self, input_data=[], batch_size=10):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)